client/src/helpers/APICalls/getMessagesByConvoId.ts

```diff
@@ -0,0 +1,18 @@
+import { FetchOptions } from '../../interface/FetchOptions';
+import { Messages } from '../../interface/Message';
+
+interface Props {
+  convoID: string | undefined;
+}
+
+export default async function searchUsers({ convoID }: Props): Promise<Messages> {
```

Comment:
This looks like a function to get messages in a conversation, maybe rename it?

Reply:
implemented request changes.